pkg/proxy/request.go

```diff
@@ -330,18 +316,22 @@ func getDuration(r *http.Request) (time.Duration, error) {
 	return time.ParseDuration(val)
 }
 
-func getImage(r *http.Request, role string) apisLoadTestV1.ImageDetails {
-
+func getImage(r *http.Request, role string) (apisLoadTestV1.ImageDetails, error) {
 	imageStr := r.FormValue(role)
 
+	imageRegex := regexp.MustCompile("^.*:.*$|^$")
```

Comment:
could you move this to a package `var` like the others in line 27 ?
the regexp compiling is a resource expensive process but it need to be done only once.

Reply:
Thanks for noticing it! Moved to vars.